charts/timescaledb-single/Chart.yaml

```
@@ -4,7 +4,7 @@
 apiVersion: v1
 name: timescaledb-single
 description: 'TimescaleDB HA Deployment.'
-version: 0.3.0
+version: 0.4.0
```

We can do it in this commit, but we're normally bumping this one when we create the release commit.
```suggestion
version: 0.3.0
```

---

We also need some more documentation I guess, shouldn't be part of this PR, so created a placeholder here: https://github.com/timescale/timescaledb-kubernetes/pull/66